### DISCUSSION TITLE
10월 4주차

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#50] [Weekly meeting]
10/17 월요일 회의 주제 목록

### sx prop 의 제한을 둘 것인지

- 제한 O -> 디자인에 의한 일관성
- 제한 X -> 예외 발생의 가능성

**Comment by hemudi** [comment]
### 프로젝트 소개글 작성
현재 프로젝트 소개글에 어떤 내용이 들어가면 좋을지 주제만 정리해둔 상태
협조 요청

**Comment by jindonyy** [comment]
## CSS Convention
- [NHN css convention](http://www.tutor4u.co.kr/data/member/test16.pdf)
- [Mozilla 컨벤션](http://milooy.github.io/TIL/CSS/css-property-order.html#intro)
- [추천하고 싶은 규칙](https://jeonghakhur.gitbooks.io/frontend-coding-convention/content/css/)
